libfaketime/__init__.py

```diff
@@ -101,7 +101,7 @@ def end_callback(instance):
 
 
 class fake_time(ContextDecorator):
-    def __init__(self, datetime_spec, only_main_thread=True):
+    def __init__(self, datetime_spec, only_main_thread=False):
```

Comment:
Let's leave this as it was so the change is backwards-compatible. Callers who need it can just manually set it.